src/rpc/core_rpc_server_commands_defs.h

```diff
@@ -2981,7 +2981,13 @@ namespace cryptonote
   {
     struct request
     {
+      std::string version_major; // Storage Server Major version
```

Comment:
Could we make the request send the version as 3 integers?

Reply:
Don't see why we couldn't. Updated.